packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -276,11 +252,34 @@ export default class SalesforceCRMService implements CRM {
     }
   }
 
-  async getContacts(email: string | string[]) {
+  async getContacts(email: string | string[], includeOwner?: boolean) {
     const conn = await this.conn;
     const emails = Array.isArray(email) ? email : [email];
-    const soql = `SELECT Id, Email FROM Contact WHERE Email IN ('${emails.join("','")}')`;
+    // TODO get the user associated with the contact. Return the email
+    const soql = `SELECT Id, Email, OwnerId FROM Contact WHERE Email IN ('${emails.join("','")}')`;
     const results = await conn.query(soql);
+
+    if (!results || !results.records.length) return [];
+
+    if (includeOwner) {
```

Comment:
We have to make different calls to Salesfoce for the owner information. Let's hide this logic behind a param to only make those calls if we need to.